site/source/tutorials/1010_containers_and_injecting_state/index.md

```diff
@@ -59,55 +57,58 @@ Now that we have moved state management to a dedicated module, we need a way to
 
 ## Injectors
 
-{% task 'Register an `Injector` that will allow state to be injected into widgets.' %}
+{% task 'Register an injector factory that will allow state to be injected into widgets.' %}
 
 Currently, the application's `main` module is only responsible for creating the `Projector`, which provides the bridge between the application code and the DOM.
 
 {% include_codefile 'demo/initial/biz-e-corp/src/main.ts' %}
 
 Now, we need to:
 
-1. Update the `main` module to construct and populate an instance of `ApplicationContext`
-2. Create a `registry` and then define the `ApplicationContext` as an injector in the registry
-3. To make the `registry` available within the widget tree, we need to pass the `registry` as a property to the `projector`
+1. Create a `registry` and then define an injector factory that creates the `ApplicationContext` passing the invalidator and initial state. Returning a function that returns this instance.
+2. To make the `registry` available within the widget tree, we need to pass the `registry` as a property to the `projector`
 
 {% instruction 'Import the `ApplicationContext` module and add this code to the `main` module:' %}
 
-{% include_codefile 'demo/finished/biz-e-corp/src/main.ts' lines:4,9-28 %}
+{% include_codefile 'demo/finished/biz-e-corp/src/main.ts' lines:4 %}
 
 {% aside 'Loading data' %}
 In a real-world application, this data would probably be loaded via a call to a web service or a local data store. To learn more, take a look at the [stores tutorial](../comingsoon.html).
 {% endaside %}
 
 The state stored in the `ApplicationContext` is the same data that was used in the previous version of the `App` module to initialize the `WorkerProperties`, but it is now decoupled into an isolated module that helps to understand and maintain the application. In general, the `main` module of an application should be concerned with initializing application-wide state. Also, as previously mentioned, the `App` class only needed to manage the `WorkerProperties` state so that it could coordinate change to its children.
 
-Now that we have the `ApplicationContext` instance, let's use it within our widgets. This is done by making the `ApplicationContext` available to the widgets via a `registry`.
+Now we need to create the registry, injector factory that creates and returns the `ApplicationContext` instance injector, and finally make the registry available to the widget tree.
```

Comment:
Since create only applies to the first two listed items it might be clearer if it's repeated:
"create the registry, create the injector factory that ..., and finally make the registry available..."

Reply:
Done